finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/service/H2ReqRepFrame.scala

```diff
@@ -1,9 +1,10 @@
 package com.twitter.finagle.buoyant.h2
 package service
 
-import com.twitter.finagle.buoyant.h2.service.H2ReqRep.{FinalFrame, RepAndFrame}
-import com.twitter.finagle.service.ReqRep
-import com.twitter.util.{Return, Throw, Try}
+import com.twitter.finagle.buoyant.h2.service.H2ReqRepFrame.{FinalFrame, RepAndFrame}
+import com.twitter.util.Try
+
+case class H2ReqRep(request: Request, response: Try[Response])
```

Comment:
Curious to know if there's any particular rationale for not just using (or extending) `ReqRep` here?